yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -202,7 +202,7 @@ def _probe_version(self):
 
     @property
     def available(self):
-        return self.basename is not None
+        return self._ffmpeg_location.get() or self.basename is not None
```

Comment:
so we always get bool
```suggestion
        return bool(self._ffmpeg_location.get()) or self.basename is not None
```